Model/Event/Transport/Adapter/GraphQL.php

```diff
@@ -414,11 +414,17 @@ protected function afterRequest(array $event, string $body)
      *
      * @return string
      */
-    private static function generateRequestId(): string
+    private static function generateRequestId(array $event): string
     {
-        // Use md5 to ensure that the resulting ID is at least 20 characters.
-        // Note this ID does not need to be cryptographic random hence the use of md5 & uniqid etc.
-        return md5(uniqid(strval(rand()), true));
+        $store = "-no-event-";
+        $entity = "-no-event-";
+        if (!empty($event)) {
+            $store = empty($event['store_id']) ? "-absent-" : $event['store_id'];
+            $entity = empty($event[ResourceModel::ENTITY_ID]) ? "-absent-" : $event[ResourceModel::ENTITY_ID];
+        }
+        // Request IDs need to be more than 20 characters long.
+        // uniqid("", true) returns at least a 23 character long string.
+        return uniqid( "S$store" . "E$entity" . "-" , true);
```

Comment:
```suggestion
        $store = "no-event";
        $entity = "no-event";
        if (!empty($event)) {
            $store = empty($event['store_id']) ? "absent" : $event['store_id'];
            $entity = empty($event[ResourceModel::ENTITY_ID]) ? "absent" : $event[ResourceModel::ENTITY_ID];
        }
        // Request IDs need to be more than 20 characters long.
        // uniqid("", true) returns at least a 23 character long string.
        return uniqid( "S=$store" . ',' . "E=$entity" . ',' , true);
```

Would this format make the request ID a little easier to parse via a regex (say `^S=([^,]*),E=([^,]*)`) and/or `cut`?.